styles/prosilver/template/toplist_body.html

```diff
@@ -1,122 +1,122 @@
-<!-- INCLUDE overall_header.html -->
+{% include 'overall_header.html' %}
 
-<h2 class="solo">{L_TOPLIST}</h2>
+<h2 class="solo">{{ lang('TOPLIST') }}</h2>
 
-<!-- IF .pagination -->
+{% if pagination %}
 <div class="action-bar top">
 	<div class="pagination">
-		<!-- INCLUDE pagination.html -->
+		{% include 'pagination.html' %}
 	</div>
 </div>
-<!-- ENDIF -->
+{% endif %}
 
-<!-- IF S_THANKS_POST_REPUT_VIEW and not S_FULL_TOPIC_RATING and not S_FULL_FORUM_RATING -->
-<h3 class="solo"><a href="{U_SEARCH_POST}" class="forumtitle">{L_RATING_TOP_POST}</a></h3>
+{% if S_THANKS_POST_REPUT_VIEW and not S_FULL_TOPIC_RATING and not S_FULL_FORUM_RATING %}
+<h3 class="solo"><a href="{{ U_SEARCH_POST }}" class="forumtitle">{{ lang('RATING_TOP_POST') }}</a></h3>
 	<div class="forumbg forumbg-table">
 		<div class="inner">
 
 			<table class="table1" cellspacing="1">
 				<thead>
 				<tr>
-					<th class="topictitle">{L_RATING_POST}</th>
-					<th class="rating">{L_RECEIVED}</th>
-					<th class="name">{L_AUTHOR}</th>
-					<th class="rating">{L_REPUT}</th>
+					<th class="topictitle">{{ lang('RATING_POST') }}</th>
+					<th class="rating">{{ lang('RECEIVED') }}</th>
+					<th class="name">{{ lang('AUTHOR') }}</th>
+					<th class="rating">{{ lang('REPUT') }}</th>
 				</tr>
 				</thead>
 				<tbody>
-		<!-- BEGIN toppostrow -->
-				<tr class="<!-- IF toppostrow.S_ROW_COUNT is even -->bg1<!-- ELSE -->bg2<!-- ENDIF -->">
-					<td><a href="{toppostrow.POST_ID}" class="post">{toppostrow.MINI_POST_IMG}{toppostrow.POST_DATE}&nbsp;{toppostrow.POST_SUBJECT}</a><br />{toppostrow.MESSAGE}</td>
-					<td>{toppostrow.POST_THANKS}</td>
-					<td>{toppostrow.POST_AUTHOR}</td>
-					<td><span style="display: block;">{L_REPUT}{L_COLON}&nbsp;{toppostrow.POST_REPUT}
-						<!-- IF toppostrow.S_THANKS_REPUT_GRAPHIC -->
-						<span style="display: block; width: {toppostrow.THANKS_REPUT_GRAPHIC_WIDTH}; height: {toppostrow.THANKS_REPUT_HEIGHT}; background: url({toppostrow.THANKS_REPUT_IMAGE_BACK}); background-repeat: repeat-x;"><span style="display: block; height: {toppostrow.THANKS_REPUT_HEIGHT}; width: {toppostrow.POST_REPUT}; background: url({toppostrow.THANKS_REPUT_IMAGE}); background-repeat: repeat-x;">&nbsp;</span></span>
-						<!-- ENDIF -->
+		{% for tprow in loops.toppostrow %}
```

Comment:
I know it isn't required for 3.2, but I wasn't sure if this repository worked for 3.1.x, and if it did I wanted to have it be compatible with both.  If this repo is only for 3.2 then I'll remove all of that.